### DISCUSSION TITLE
Working with jobs

### DISCUSSION THREAD

**Discussion by yuval-airis** [dagster-io/dagster#18885] [Q&A]
I have been using asset graphs for a while now with success, where I simply defined a few assets, and they connected to a graph by the dependencies in the function parameters.

However, now I try to define a job explicitly for the first time and try to run it [with the graphQL API](https://docs.dagster.io/concepts/webserver/graphql-client#submitting-a-job-run), but I keep failing and I can't tell what the issue is from the error.

On the graphQL call side I got:
```
        response = dagster_client.submit_job_execution(
            job_name="do_stuff",
            op_selection=["files", "print_files"]
        )
```

While in the ops graph side I have:
```
@asset
def print_files(files):
    print("X")
    print(files)
    print("Y")

@asset
def files():
    files = ['myName']
    new_files = [filename + filename for filename in files]

    return new_files


@job
def do_stuff():
    print("Step 1")
    duplicate_files = files()
    print(f"Step 2 - {duplicate_files}")
    print_files(duplicate_files)
```

Which throws an exception I can't decipher a lot from:
```
DagsterGraphQLClientError('Exception occured during execution of query \n\nquery {\n  repositoriesOrError {\n    __typename\n    ... on RepositoryConnection {\n      nodes {\n        name\n        location {\n          name\n        }\n        pipelines {\n          name\n        }\n      }\n    }\n    ... on PythonError {\n      message\n    }\n  }\n}\n\n with variables \nNone\n')
Exception occured during execution of query
```

What does this error message mean, and how may I make the trigger actually calculate an op (or better yet, a graph of ops) for me?

**Comment by johannkm** [comment]
Could you check the logs in your dagster server process?
